pom.xml

```diff
@@ -55,21 +82,35 @@
 		<dependency>
 			<groupId>org.powermock</groupId>
 			<artifactId>powermock-api-mockito</artifactId>
-			<version>1.6.5</version>
+			<version>${powermock.version}</version>
 			<scope>test</scope>
 		</dependency>
 		<dependency>
 			<groupId>org.powermock</groupId>
 			<artifactId>powermock-module-junit4</artifactId>
-			<version>1.6.5</version>
+			<version>${powermock.version}</version>
 			<scope>test</scope>
 		</dependency>
 	</dependencies>
 
 	<build>
 		<plugins>
+			<!-- sets the Java compiler version targeted by Maven -->
 			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-compiler-plugin</artifactId>
+				<version>3.8.1</version>
+				<configuration>
+					<source>${maven.compiler.source}</source>
```

Comment:
I _think_ that if maven.compiler.source and ...target are defined as properties, maven is clever enough to use the properties and there is no need to define this plugin with these values. See: https://maven.apache.org/plugins/maven-compiler-plugin/examples/set-compiler-source-and-target.html

Reply:
Okay, I've removed the property values and just have the values set in the compiler plugin itself so it the compiler plugin version and options are explicit

Reply:
Curious why that and not remove the compiler plugin? If only the properties are left, the version is still explicit.

Reply:
I could add the properties and remove the compiler plugin.  I was just thinking that the other projects also have the compiler plugin specified, it sets the version of the plugin instead of defaulting to the latest version to allow for stable/repeatable builds, and if we want to customize its configuration we can.

I don't have strong feelings about keeping the compiler plugin instead of the properties, though.  Do you think the properties are preferable?

I'd just need to go into the other projects POMs and swap out the compiler plugin for the maven compiler properties.

Reply:
No, I don't have any strong feelings, I was just surprised. I thought you would have gone the other way and simplified the POM by removing the plugin and just leaving the properties instead of leaving the compiler plugin with a specific version.

I don't think it matter strongly either way. Like I said, I was just surprised by which solution you chose. ;)

Reply:
Yeah, I was considering both options.  I just thought it was nice (though a bit more verbose) to keep the compiler options in the build section with the other plugins and have the specific version of the compiler plugin nailed down.

Reply:
Hehe yeah I think I usually do it the other way, but both are fine. :)